env.yml

```diff
@@ -26,4 +26,4 @@ dependencies:
 
     # additional dependencies of the file directory support additions
     # install from GitHub repository
-    - "https://raw.githubusercontent.com/EPICS-synApps/support/master/utils/mdaPythonUtils/mdalib-2022.post9-py2.py3-none-any.whl"
\ No newline at end of file
+    - https://github.com/EPICS-synApps/support/blob/master/utils/mdaPythonUtils/mda.py
```

Comment:
```suggestion
    - https://raw.githubusercontent.com/EPICS-synApps/support/master/utils/mdaPythonUtils/mdalib-2022.post9-py2.py3-none-any.whl
```

Reply:
Need the raw content, otherwise it will download the HTML-formatted page.

Reply:
Isn't it what I had initially. I misunderstood your comment [here](https://github.com/BCDA-APS/mdaviz/issues/52#issuecomment-1922273161)

Reply:
Yes, it is.